.github/workflows/update_phishing_detection_data.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Phishing Detection Datasets
+on:
+  push:
+jobs:
+  update_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
checkout@v3 is outdated - let's checkout@v4 to avoid GHA posting warnings under workflows.